HISTORY.rst

```diff
@@ -42,6 +42,25 @@ Unreleased Changes
       in the list of recent runs.
 * NDR
     * Added parameters to the sample data to support nitrogen calculations.
+* SDR
+    * We have made a significant update to the SDR model's outputs:
+      * Legacy outputs ``sed_retention_index.tif`` and ``sed_retention.tif``
+        have been removed from the model.
+      * Two new output rasters have been added:
+        * ``total_retention.tif``, indicating vegetation's contribution to
```

Comment:
In the PEP document it says the 2 new outputs should be:
1) Avoided Export
2) Avoided Erosion

Is `total_retention.tif` the inverse of `avoided_export`?

Reply:
Okay, I also see the PEP refer to Avoided Export as Total Retention. I guess it would just be nice to be consistent with whatever the UG goes with.

Reply:
Yeah, they're synonyms.  I re-read the slack thread at https://stanford-natcap.slack.com/archives/CECFUU1DW/p1649109737853559?thread_ts=1647968772.949939&cid=CECFUU1DW and settled on:

* "Avoided Export" instead of "Total Retention"
* "Avoided local erosion" instead of "Avoided erosion"